index.html

```diff
@@ -29,31 +29,185 @@
     </header>
 
     <!-- main section -->
-    <main class="main-section">
-      <div class="content">
-        <p class="author">
-          I’m Ismail <br />
-          Glad to see <br />
-          you!
-        </p>
-        <p class="article">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
+    <main>
+      <!-- navbar headline features -->
+      <div class="headline-section">
+        <div class="content">
+          <p class="author">
+            I’m Ismail <br />
+            Glad to see <br />
+            you!
+          </p>
+          <p class="article">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+
+        <div class="connect">
+          <p>LET’S CONNECT</p>
+        </div>
+
+        <div class="social-icons">
+          <span class="iconify" data-icon="iconoir:github"></span>
+          <span class="iconify" data-icon="eva:linkedin-fill"></span>
+          <span class="iconify" data-icon="fontisto:angelist"></span>
+          <span class="iconify" data-icon="ei:sc-twitter"></span>
+          <span class="iconify" data-icon="bx:bxl-medium"></span>
+        </div>
+      </div>
+
+      <!-------------------------------------- middle-section ---------------------------------------->
+      <!-- tonic section 1 -->
+      <div class="tonic-section1">
+        <div class="tonic-image1"></div>
+        <h2>Tonic</h2>
+        <span class="canopy">CANOPY</span>
+        <span class="dev">• Back End Dev • 2015</span>
+        <div class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </div>
+
+        <button class="html">html</button>
+        <button class="css">css</button>
+        <button class="javascript">javascript</button>
+
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be wrapped in an unordered list `ul` tag and each technology used should be a `li` tag.

- [ ] Please make sure to follow the design guidelines for the margins (top, bottom, left, right) defined in the design, also please make sure that your text color, background color, and padding are matching the Figma design.